compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -3463,6 +3465,28 @@ private STNode parseFuncCall(STNode identifier) {
         return STNodeFactory.createFunctionCallExpressionNode(identifier, openParen, args, closeParen);
     }
 
+    /**
+     * Parse error constructor expression.
+     *
+     * @return Error constructor expression
+     */
+    private STNode parseErrorConstructorExpr() {
+        STNode errorKeyword = parseErrorKeyword();
+        STNode openParen = parseOpenParenthesis();
+        STNode args = parseArgsList();
+        STNode closeParen = parseCloseParenthesis();
+        return STNodeFactory.createErrorConstructorExpressionNode(errorKeyword, openParen, args, closeParen);
+    }
+
+    /**
+     * Parse error keyword.
+     *
+     * @return Error keyword
+     */
+    private STNode parseErrorKeyword() {
```

Comment:
I think theres a method with the similar method already

Reply:
Removed